### PR TITLE
Cg 22

### PR DESCRIPTION
**Pull request by cgaddis36**
Added in tests for edge-case testing.
Added in test for if an email is entered in the profile edit is already taken in the database. everything passing

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def update
       redirect_to '/profile'
     else
       flash[:error] = @user.errors.full_messages.to_sentence
-      render :edit
+      redirect_to '/profile/edit'
```
Comment (adumortier) [inline app/controllers/users_controller.rb:40]: why did this line need to be changed?

File: spec/features/users/edit_profile_spec.rb [hunk 2]
```diff
@@ -43,4 +53,53 @@
...
+    fill_in :state, with: "Hawaii"
+    fill_in :zip_code, with: "12222"
+    fill_in :email, with: @default_user2.email
+
+    click_on("Update Profile")
+
+    expect(current_path).to eq("/profile/edit")
+    expect(page).to have_content("Email has already been taken")
+  end
+  it "has flash messages when you erase the required user attributes" do
```
Comment (adumortier) [inline spec/features/users/edit_profile_spec.rb:81]: nice test! looks pretty robust now!

**Review by adumortier — APPROVED** [review]
Tried to break the edit page from `rails s` but couldn't :) looks good to me!

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (KellyIB) [reply app/controllers/users_controller.rb:40]: was the render not working?

File: spec/features/users/edit_profile_spec.rb (hunk 2 above)
Comment (KellyIB) [reply spec/features/users/edit_profile_spec.rb:81]: Looks good.

**Review by KellyIB — APPROVED** [review]
